### ISSUE TITLE
[QA] docker images have no support for PDF or SVG preview providers

### ISSUE THREAD

**Issue by jnweiger** [owncloud/core#41185]
Seen in 10.14.0-rc.1 (but not a regression)

* The self-compiled imagick in our docker images reports via phpinfo():
```
Imagick compiled with ImageMagick version => ImageMagick 7.1.1-27 Q16-HDRI x86_64 9c64c662f:20240120 https://imagemagick.org
Imagick using ImageMagick library version => ImageMagick 7.1.1-27 Q16-HDRI x86_64 9c64c662f:20240120 https://imagemagick.org
ImageMagick copyright => (C) 1999 ImageMagick Studio LLC
ImageMagick release date => 2024-01-20
ImageMagick number of supported formats:  => 213
ImageMagick supported formats => 3G2, 3GP, AAI, APNG, ART, ASHLAR, AVI, AVIF, AVS, BAYER, BAYERA, BGR, BGRA, BGRO, BIE, BMP, BMP2, BMP3, BRF, CAL, CALS, CANVAS, CAPTION, CIN, CIP, CLIP, CMYK, CMYKA, CUBE, CUR, CUT, DATA, DCM, DCX, DDS, DFONT, DPX, DXT1, DXT5, EPS2, EPS3, EPT, EPT2, EPT3, FARBFELD, FAX, FF, FILE, FITS, FL32, FLV, FRACTAL, FTP, FTS, FTXT, G3, G4, GIF, GIF87, GRADIENT, GRAY, GRAYA, GROUP4, HALD, HDR, HEIC, HEIF, HISTOGRAM, HRZ, HTM, HTML, HTTP, HTTPS, ICB, ICO, ICON, INFO, INLINE, IPL, ISOBRL, ISOBRL6, JBG, JBIG, JNG, JNX, JPE, JPEG, JPG, JPS, JSON, JXL, KERNEL, LABEL, M2V, M4V, MAC, MAP, MASK, MAT, MATTE, MIFF, MKV, MNG, MONO, MOV, MP4, MPC, MPEG, MPG, MPO, MSL, MTV, MVG, NULL, ORA, OTB, OTF, PAL, PALM, PAM, PATTERN, PBM, PCD, PCDS, PCL, PCT, PCX, PDB, PES, PFA, PFB, PFM, PGM, PGX, PHM, PICON, PICT, PIX, PJPEG, PLASMA, PNG, PNG00, PNG24, PNG32, PNG48, PNG64, PNG8, PNM, PPM, PS2, PS3, PSB, PSD, PTIF, PWP, QOI, RADIAL-GRADIENT, RAS, RGB, RGB565, RGBA, RGBO, RGF, RLA, RLE, SCR, SCT, SFW, SGI, SHTML, SIX, SIXEL, SPARSE-COLOR, STEGANO, STRIMG, SUN, TEXT, TGA, THUMBNAIL, TIFF, TIFF64, TILE, TIM, TM2, TTC, TTF, TXT, UBRL, UBRL6, UIL, UYVY, VDA, VICAR, VID, VIFF, VIPS, VST, WBMP, WEBM, WMV, WPG, X, XBM, XC, XCF, XPM, XPS, XV, XWD, YAML, YCbCr, YCbCrA, YUV
```

* PDF and SVG support should come from imagick according to https://doc.owncloud.com/server/next/admin_manual/configuration/files/previews_configuration.html#preview-format-requirements
* PDF and SVG are not present in the above list, and thus previews cannot be enabled. BAD

Expected behaviour:
 * PDF and SVG should show previews when started with e.g.
 `docker run --rm -ti -p 8080:8080 -e 'OWNCLOUD_ENABLED_PREVIEW_PROVIDERS=OC\Preview\PDF,OC\Preview\SGI,OC\Preview\Heic,OC\Preview\PNG,OC\Preview\JPEG,OC\Preview\WEBP,OC\Preview\GIF,OC\Preview\BMP,OC\Preview\XBitmap,OC\Preview\MP3,OC\Preview\TXT,OC\Preview\MarkDown' owncloud/server:10.14.0-rc.1`

**Comment by DeepDiver1975** [comment]
there is a config file with imagick where formats can be enabled ...

**Comment by DeepDiver1975** [comment]
https://stackoverflow.com/questions/52998331/imagemagick-security-policy-pdf-blocking-conversion

**Comment by jnweiger** [comment]
Added details to https://github.com/owncloud/docs-server/pull/1269

Tested both cases: imagic with and without compiled PDF support. Editing the policy only helps in the first case.

**Comment by DeepDiver1975** [comment]
I never had to compile imagick extension myself .... one day I need to understand why you do this ...

**Comment by jnweiger** [comment]
Maybe today: We could not find an imagick package with heic support in ubuntu 20.04, when we had to add this to our docker. Then we decided to build our own.

Probably the docker images should be rebased to something much newer (php-7.4 :see_no_evil: ), but that was not the plan at that time ...

**Comment by pako81** [comment]
This is already documented: https://doc.owncloud.com/server/next/admin_manual/configuration/files/previews_configuration.html#notes-for-pdf-preview-generation

**Comment by iasdeoupxe** [comment]
Shouldn't this go to https://github.com/owncloud-docker/server/issues as this doesn't look ownCloud server specific but docker specific?

**Comment by jnweiger** [comment]
We deployed fixed docker images for 10.13.4 and 10.14.0. Please try e.g.  

```
docker pull owncloud/server:10.14.0
docker run  -ti -p 8080:8080 -e 'OWNCLOUD_ENABLED_PREVIEW_PROVIDERS=OC\Preview\PDF,OC\Preview\SGI,OC\Preview\Heic,OC\Preview\PNG,OC\Preview\JPEG,OC\Preview\WEBP,OC\Preview\GIF,OC\Preview\BMP,OC\Preview\XBitmap,OC\Preview\MP3,OC\Preview\TXT,OC\Preview\MarkDown,OC\Preview\SVG'  owncloud/server:10.14.0
```
Confirmed fixed. All kinds of previews work nicely for me.
